packages/node_modules/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1012,11 +1003,12 @@ export const AVALIABLE_RESOLUTIONS = {
 
 export const VIDEO_RESOLUTIONS = {
   [QUALITY_LEVELS.LOW]: AVALIABLE_RESOLUTIONS['360p'],
-
   [QUALITY_LEVELS.MEDIUM]: AVALIABLE_RESOLUTIONS['480p'],
-
   [QUALITY_LEVELS.HIGH]: AVALIABLE_RESOLUTIONS['720p'],
```

Comment:
Do we need to changes these as 
```
LOW : 480p
MEDIUM : 720p
HIGH : 1080p
```

Reply:
Yes, @rajeshtezu90, we will have all 4 option for local, for clarification let's say,
remote will have ['LOW', 'MEDIUM', 'HIGH']; where low will be 480, Medium to 720p and high to 1080p. 
and for local will have ['360p', '480p', '720p', '1080p'];

Reply:
+1

Reply:
Done.